http_post.go

```diff
@@ -27,6 +30,30 @@ func newHTTPPostNode(et *ExecutingTask, n *pipeline.HTTPPostNode, l *log.Logger)
 		bp:   bufpool.New(),
 		url:  n.Url,
 	}
+
+	if n.Url != "" {
+		c := httppost.HandlerConfig{}
+		c.URL = n.Url
+		h := et.tm.HTTPPostService.Handler(c, l)
```

Comment:
The type change is also useful here since you can return the concrete type so you do not have to type assert it. Or event add a new method for getting Endpoints.